scripts/post_build.d/build_nginx.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
Remove kabanero references.

Reply:
What do you suggest as an alternative - should this be "Appsody"

Reply:
Done